modules/ipi-install-node-requirements.adoc

```diff
@@ -37,6 +37,11 @@ Do not deploy a cluster with only one worker node, because the cluster will depl
 * *Network interfaces:* Each node must have at least one network interface for the routable `baremetal` network. Each node must have one network interface for a `provisioning` network when using the `provisioning` network for deployment. Using the `provisioning` network is the default configuration.
 
 * *Unified extensible firmware interface (UEFI):* Installer-provisioned installation requires UEFI boot on all {product-title} nodes when using IPv6 addressing on the `provisioning` network. In addition, UEFI Device PXE Settings must be set to use the IPv6 protocol on the `provisioning` network NIC, but omitting the `provisioning` network removes this requirement.
++
```

Comment:
I tried to determine why all of this text is shown in italics in this PR. The italics start at [line 6](https://github.com/openshift/openshift-docs/pull/47205/files#diff-0325fc3eff17d0b0c822aa05de994d2b0fc909b0cdcbfb7685c4bd438a3c24b1R6). Looking at that line, the ID has a spurious `'"` entry. Although it doesn't seem to impact the rendered build, it would be useful to resolve this. Can you please update that ID to the following?:

> `[id="node-requirements_{context}"]`